playbook.md

```diff
@@ -30,7 +30,7 @@ We think that it's very important to have a talented team if we're going to succ
 We are helpful, useful and expert. We give practical and pragmatic advice to each other, and to our clients.
 
 #### Honest
-We are honest, trustworthy and straightforward. We give plain-spoken, frank, accurate feedback and advice, and we never mislead or obfuscate.
+We are honest, trustworthy and straightforward. We give plain-spoken, frank, accurate feedback and advice, and we never mislead or wilfully confuse.
```

Comment:
I think `obscure the issue` probably covers both bases, anyway. So I'm happy with `we never mislead or obscure the issue at hand`.